blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DatastaxLocatorIO.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2016 Rackspace.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.rackspacecloud.blueflood.io.datastax;
+
+import com.codahale.metrics.Timer;
+import com.datastax.driver.core.*;
+import com.datastax.driver.core.querybuilder.Insert;
+import com.datastax.driver.core.querybuilder.QueryBuilder;
+import com.datastax.driver.core.querybuilder.Select;
+import com.rackspacecloud.blueflood.cache.TenantTtlProvider;
+import com.rackspacecloud.blueflood.io.CassandraModel;
+import com.rackspacecloud.blueflood.io.Instrumentation;
+import com.rackspacecloud.blueflood.io.LocatorIO;
+import com.rackspacecloud.blueflood.types.Locator;
+import com.rackspacecloud.blueflood.utils.Util;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import static com.datastax.driver.core.querybuilder.QueryBuilder.bindMarker;
+import static com.datastax.driver.core.querybuilder.QueryBuilder.eq;
+import static com.datastax.driver.core.querybuilder.QueryBuilder.ttl;
+
+/**
+ * This class uses the Datastax driver to read/write locators from
+ * Cassandra metrics_locator Column Family.
+ */
+public class DatastaxLocatorIO implements LocatorIO {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DatastaxLocatorIO.class);
+
+    public static final String KEY = "key";
+    public static final String COLUMN1 = "column1";
+    public static final String VALUE = "value";
+
+    private PreparedStatement getValue;
+    private PreparedStatement putValue;
+
+    public DatastaxLocatorIO() {
+        createPreparedStatements();
+    }
+
+    /**
+     * Create all prepared statements use in this class for metrics_locator
+     */
+    private void createPreparedStatements()  {
+
+        // create a generic select statement for retrieving from metrics_locator
+        Select.Where select = QueryBuilder
+                .select()
+                .all()
+                .from( CassandraModel.CF_METRICS_LOCATOR_NAME )
+                .where( eq ( "key", bindMarker() ));
+        getValue = DatastaxIO.getSession().prepare( select );
+
+        // create a generic insert statement for inserting into metrics_locator
+        Insert insert = QueryBuilder.insertInto(CassandraModel.QUOTED_KEYSPACE, CassandraModel.CF_METRICS_LOCATOR_NAME)
```

Comment:
Cool!  This was the cause of the instantiate error.  Now I can get rid of that too!